src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNodeKind.cs

```diff
@@ -14,95 +14,95 @@ internal enum RegexNodeKind
         /// This is purely a representational optimization, equivalent to a <see cref="Lazyloop"/> wrapped around a <see cref="Notone"/>.
-        Notonelazy = RegexCode.Notonelazy,
+        Notonelazy = RegexOpcode.Notonelazy,
         /// <summary>A lazy loop around a character class / set, e.g. `[a-z1-9]*?` or `\w?`.</summary>
         /// <remarks>The <see cref="RegexCharClass"/> set string is specified in <see cref="RegexNode.Str"/>, the minimum number of iterations in <see cref="RegexNode.M"/>, and the maximum number of iterations in <see cref="RegexNode.N"/>.</remarks>
         /// This is purely a representational optimization, equivalent to a <see cref="Lazyloop"/> wrapped around a <see cref="Set"/>.
-        Setlazy = RegexCode.Setlazy,
+        Setlazy = RegexOpcode.Setlazy,
 
         /// <summary>An atomic loop around a specific character, e.g. `(?> a*)`.</summary>
         /// <remarks>
         /// The character is specified in <see cref="RegexNode.Ch"/>, the minimum number of iterations in <see cref="RegexNode.M"/>, and the maximum number of iterations in <see cref="RegexNode.N"/>.
         /// This is purely a representational optimization, equivalent to a <see cref="Atomic"/> wrapped around a <see cref="Oneloop"/>.
         /// </remarks>
-        Oneloopatomic = RegexCode.Oneloopatomic,
+        Oneloopatomic = RegexOpcode.Oneloopatomic,
         /// <summary>An atomic loop around anything other than a specific character, e.g. `(?>.*)` when not in <see cref="RegexOptions.Singleline"/> mode.</summary>
         /// <remarks>
         /// The character is specified in <see cref="RegexNode.Ch"/>, the minimum number of iterations in <see cref="RegexNode.M"/>, and the maximum number of iterations in <see cref="RegexNode.N"/>.
         /// This is purely a representational optimization, equivalent to a <see cref="Atomic"/> wrapped around a <see cref="Notoneloop"/>.
         /// </remarks>
-        Notoneloopatomic = RegexCode.Notoneloopatomic,
+        Notoneloopatomic = RegexOpcode.Notoneloopatomic,
         /// <summary>An atomic loop around a character class / set, e.g. `(?>\d*)`.</summary>
         /// <remarks>
         /// The <see cref="RegexCharClass"/> set string is specified in <see cref="RegexNode.Str"/>, the minimum number of iterations in <see cref="RegexNode.M"/>, and the maximum number of iterations in <see cref="RegexNode.N"/>.
         /// This is purely a representational optimization, equivalent to a <see cref="Atomic"/> wrapped around a <see cref="Setloop"/>.
         /// </remarks>
-        Setloopatomic = RegexCode.Setloopatomic,
+        Setloopatomic = RegexOpcode.Setloopatomic,
 
         /// <summary>A backreference, e.g. `\1`.</summary>
         /// <remarks>The capture group number referenced is stored in <see cref="RegexNode.M"/>.</remarks>
-        Backreference = RegexCode.Ref,
+        Backreference = RegexOpcode.Ref,
```

Comment:
NIT: While we are at it, should we rename this to `Backreference` for consistency and for readability?

Reply:
Good suggestion, will do.  I'll do that in this PR if I need to restart CI for some reason, otherwise subsequently.